lib/widgets/recent_dm_conversations.dart

```diff
@@ -60,7 +66,28 @@ class _RecentDmConversationsPageBodyState extends State<RecentDmConversationsPag
             narrow: narrow,
             unreadCount: unreadsModel!.countInDmNarrow(narrow),
           );
-        }));
+        })),
+      floatingActionButton: Container(
+        constraints: const BoxConstraints(
+          minWidth: 137,
+          minHeight: 48),
+        child: FloatingActionButton.extended(
+          elevation: 0,
+          onPressed: () {
+            showNewDmSheet(context);
+          },
+          extendedPadding: const EdgeInsetsDirectional.fromSTEB(16, 12, 20, 12),
+          icon: const Icon(Icons.add, size: 24),
+          label: Text(
+            zulipLocalizations.newDmFabButtonLabel,
+            style: const TextStyle(fontSize: 20, height: 24 / 20)
+              .merge(weightVariableTextStyle(context, wght: 500))),
+          backgroundColor: designVariables.fabBg,
```

Comment:
Trying out the PR, this color doesn't look right.

It looks like the screenshots in the PR description agree with what I see running it (good), but don't agree on this color with what's in Figma.